### ISSUE TITLE
NER fine tune mismatch size issue

### ISSUE THREAD

**Issue by sersoage** [ThilinaRajapakse/simpletransformers#255]
Hi, i trained a NER model with a different set of labels, which were passed down when the model was first created. I trained the model, saved the checkpoints and best model and prediction was working fine.
But when I try to load the save model I get the following error:
Error(s) in loading state_dict for BertForTokenClassification:
	size mismatch for classifier.weight: copying a param with shape torch.Size([2, 768]) from checkpoint, the shape in current model is torch.Size([9, 768]).
	size mismatch for classifier.bias: copying a param with shape torch.Size([2]) from checkpoint, the shape in current model is torch.Size([9]).
It seems that the model head is "stuck" with the original labels, is there a way to avoid this error?

**Comment by ThilinaRajapakse** [comment]
You'll need to load the custom labels when you load the saved model as well.

**Comment by sersoage** [comment]
Thanks! closing issue!

**Comment by sersoage** [comment]
Awesome response!
